src/test/java/nl/tudelft/ti2206/group9/gui/EndToEndTest.java

```diff
@@ -49,7 +51,7 @@ public void start(final Stage primaryStage) {
 	}
 
 	@Test
-	public void test() throws IOException {
+	public void test() throws IOException, MediaException {
```

Comment:
Why is the E2E throwing an exception? Shouldn't these be caught in the AudioPlayer?

<documentation